src/targets/gpu/include/migraphx/gpu/convolution.hpp

```diff
@@ -163,6 +167,20 @@ struct miopen_convolution
         auto w_desc                = make_tensor(reshape_if_1d(inputs[1]), int8_x4_format);
         auto y_desc                = make_tensor(reshape_if_1d(output_shape));
         std::size_t workspace_size = 0;
+
+        // TODO:  Using an environment variable to disable convolution tuning while still allowing
+        // other (i.e. GEMM) tuning with --exhaustive-tune, is a workaround for a problem in which
```

Comment:
It would be better to just add a env variable to enable GEMM tuning even when the exahustive flag is not passed.

Reply:
Done.  I've refactored this so that instead of specially DISabling tuning for convolutions, user can specially ENable it for GEMMs by setting MIGRAPHX_ENABLE_GEMM_TUNING.  See  src/targets/gpu/include/migraphx/gpu/gemm.hpp:142